src/trimmer.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'base'
+
+class TrimmerDecorator < BaseDecorator
+  def initialize(nameable)
+    super(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    return @nameable.correct_name.slice(0, 10) if @nameable.correct_name.length > 10
+
+    @nameable.correct_name
```

Comment:
- To make your code easier to read and more concise, I suggest using this one line `@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`